NEWS.md

```diff
@@ -0,0 +1,122 @@
+# serofoi 0.1.0
+
+## New features
+
+* Data simulation functions from time-varying or age-varying _force of infection_ trends. The following is an example to simulate from a constant (in time) _force of infection_:
+```
+foi_sim_constant <- rep(0.02, 50)
+
+serodata_constant <- generate_sim_data(
+  sim_data = data.frame(
+    age=seq(1,50),
+    tsur=2050),
+  foi=foi_sim_constant,
+  sample_size_by_age = 5
+)
+```
+To generate grouped serosurveys the function `group_sim_data` can be used:
+```
+serodata_constant <- group_sim_data(serodata_constant , step = 5) 
```

Comment:
I tried to run this example but it returns the error `R cannot find the function group_sim_data()`. Reinstalling the package from main didn't solve it. I suspect, it's because you didn't export the function. Can you check this, please? I would suggest adding the tags `@export` or `@keywords internal`.